other/upgrade.php

```diff
@@ -5761,4 +5761,63 @@ function doTheDelete(theCheck)
 			Simple Machines';
 }
 
+/**
+ * Convert MySQL (var)char ip col to binary
+ * newCol needs to be a varbinary(16) null able field
+ * return true or false
+ */
+function MySQLConvertOldIp($targetTable,$oldCol,$newCol)
+{
+	//mysql default max length is 1mb http://dev.mysql.com/doc/refman/5.1/en/packet-too-large.html
+	$max = 10000;
+	$arIp = array();
+	
+	$request = $smcFunc['db_query']('', 'DROP TABLE IF EXISTS {db_prefix}ip_table');
```

Comment:
( ! ) Fatal error: Function name must be a string in D:\www\SMF2.1\upgrade.php on line 5775